### ISSUE TITLE
Calling unknown method: craft\console\Request::getUserIP()

### ISSUE THREAD

**Issue by brianjhanson** [putyourlightson/craft-log-to-file#4]
When trying to clear the cache from the Blitz console command, we get and error about writing a log file to the disc.

It's happening over at [ L#118](https://github.com/putyourlightson/craft-log-to-file/blob/v1/src/LogToFile.php#L117-L119) in this plugin because the console request object doesn't have a getUserIp method on it and our site is set to store user IP details due to the requirements of another plugin.

I think changing those lines to: 

```php
if (Craft::$app->getConfig()->getGeneral()->storeUserIps && !Craft::$app->getRequest()->getIsConsoleRequest() ) {
    $ip = Craft::$app->getRequest()->getUserIP();
}
```

should do the trick, but not sure if that would cause any additional issues.

<details>
<summary><strong>Full Stack Trace</strong></summary>

```
Exception 'yii\base\UnknownMethodException' with message 'Calling unknown method: craft\console\Request::getUserIP()'

in /vendor/yiisoft/yii2/base/Component.php:300

Stack trace:
#0 /vendor/putyourlightson/craft-log-to-file/src/LogToFile.php(118): yii\base\Component->__call('getUserIP', Array)
#1 /vendor/putyourlightson/craft-blitz/src/Blitz.php(125): putyourlightson\logtofile\LogToFile::log('Blitz cache suc...', 'blitz', 'info')
#2 /vendor/putyourlightson/craft-blitz/src/console/controllers/CacheController.php(342): putyourlightson\blitz\Blitz->log('Blitz cache suc...')
#3 /vendor/putyourlightson/craft-blitz/src/console/controllers/CacheController.php(284): putyourlightson\blitz\console\controllers\CacheController->_output('Blitz cache suc...')
#4 /vendor/putyourlightson/craft-blitz/src/console/controllers/CacheController.php(176): putyourlightson\blitz\console\controllers\CacheController->_clearCache()
#5 [internal function]: putyourlightson\blitz\console\controllers\CacheController->actionRefresh()
#6 /vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#7 /vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#8 /vendor/yiisoft/yii2/console/Controller.php(164): yii\base\Controller->runAction('refresh', Array)
#9 /vendor/yiisoft/yii2/base/Module.php(528): yii\console\Controller->runAction('refresh', Array)
#10 /vendor/yiisoft/yii2/console/Application.php(180): yii\base\Module->runAction('blitz/cache/ref...', Array)
#11 /vendor/craftcms/cms/src/console/Application.php(87): yii\console\Application->runAction('blitz/cache/ref...', Array)
#12 /vendor/yiisoft/yii2/console/Application.php(147): craft\console\Application->runAction('blitz/cache/ref...', Array)
#13 /vendor/yiisoft/yii2/base/Application.php(386): yii\console\Application->handleRequest(Object(craft\console\Request))
#14 /craft(22): yii\base\Application->run()
#15 {main}
```

</details>

**Comment by bencroker** [comment]
Fixed with your PR, thanks!

**Comment by bencroker** [comment]
Released in version 1.2.2.
